src/layer/park.js

```diff
@@ -88,8 +88,8 @@ export const parkLabel = {
   filter: ["==", ["get", "class"], "park"],
   paint: {
     "text-color": Color.parkLabel,
-    "text-halo-blur": 1,
-    "text-halo-color": "rgba(255, 255, 255, 1)",
+    "text-halo-blur": 0.5,
+    "text-halo-color": Color.backgroundFill,
```

Comment:
The neutral background color works better than white against a park’s mint green fill, but I wonder if something closer to green would achieve an even more subtle knockout effect.

Reply:
Good idea.  I'll try it out.

Reply:
**Background fill halo:**
<img width="266" alt="background-fill-halo-green-text" src="https://user-images.githubusercontent.com/281482/205953238-4c2562a7-a481-4f8c-9b74-b02268f3d686.png">

**Park fill green halo:**
<img width="266" alt="park-fill-halo-green-text" src="https://user-images.githubusercontent.com/281482/205953433-8a59f497-f2a3-4109-b557-3220b1eaca68.png">
Ok here are some park label halo variations.

**Park fill green halo, black text:**
<img width="266" alt="park-fill-halo-black-text" src="https://user-images.githubusercontent.com/281482/205953505-1dd3f44e-3ead-46a3-8a8e-2e4dc74edfff.png">

**lighter greenish/offwhite (F0F4EC) halo:**
<img width="266" alt="F0F4EC-halo-green-text" src="https://user-images.githubusercontent.com/281482/205953826-13a37fb9-4d98-40d7-90ab-3fe07775db40.png">

Simply using the park fill green doesn't work so well in my opinion.  The green text needs more contrast with the green fill and the lighter halo helped with that.  With a darker text color that provides enough contrast the park fill green does work well.  However, I don't want to get into label text color changes in this PR.   A lighter greenish/offwhite halo is an improvement over the park fill green.  I'm not sure if it's better or worse than the background fill though.

Reply:
I've changed the park label halo color to the lighter greenish/offwhite